File: yetutil/src/main/java/net/yet/util/LogcatPrinter.java
package net.yet.util;

import android.util.Log;

import net.yet.util.xlog.XPrinter;

public class LogcatPrinter implements XPrinter {

	@Override
	public void println(int priority, String tag, String msg) {
		 Log.println(priority, tag, "" +msg);
	}

	@Override
	public void flush() {
	}

}

File: yetutil/src/main/java/yet/util/log/LogcatPrinter.kt
package yet.util.log

import android.util.Log

/**
 * Created by entaoyang@163.com on 2016-10-28.
 */

class LogcatPrinter : LogPrinter {
	override fun flush() {

	}

	override fun println(priority: Int, tag: String, msg: String) {
		Log.println(priority, tag, msg)
	}

}